CefSharp.Example/CefExample.cs

```diff
@@ -168,10 +168,11 @@ public static void Init(CefSettingsBase settings, IBrowserProcessHandler browser
 
             if (DebuggingSubProcess)
             {
-                var architecture = Environment.Is64BitProcess ? "x64" : "x86";
 #if NETCOREAPP
-                settings.BrowserSubprocessPath = Path.GetFullPath("..\\..\\..\\..\\..\\CefSharp.BrowserSubprocess\\bin.netcore\\" + architecture + "\\Debug\\netcoreapp3.1\\CefSharp.BrowserSubprocess.exe");
+                var architecture = System.Runtime.InteropServices.RuntimeInformation.ProcessArchitecture.ToString().ToLowerInvariant();
```

Comment:
I've updated the example and test projects to `.Net 4.7.2` in commit https://github.com/cefsharp/CefSharp/commit/0cdbef1d44c953beacd78384f51e1aa079710936

If I'm reading the API doc correctly `System.Runtime.InteropServices.RuntimeInformation.ProcessArchitecture` was added in `.Net 4.7.1` so we should be able to use this in all example/test projects now without needing a `if else`

Reply:
Thank you! Yes, `RuntimeInformation` was introduced in .NET Standard 2.0 and is therefore available in .NET Framework 4.7.1. I will update the code to remove the `#if`.

Reply:
Done.